### PR TITLE
Ported from feature columns to Keras layers.

### PR DESCRIPTION
**Pull request by pritamdodeja**
# High-level summary of changes

Implemented normalization, bucketization, feature crosses, and embeddings using Keras layers for both geographical and temporal features. Added explanatory comments that reinforce some of the concepts from the [class](https://www.cloudskillsboost.google/course_templates/11).  

# Motivation

Feature columns are deprecated, so I re-wrote a solution without using them.  I am going through the Machine Learning Engineer Learning Path, and this part of the class was most difficult, specifically this lab.  The use of too many complex concepts all at once was a little overwhelming, so I have tried to simplify the solution so it slowly introduces new concepts.  Specifically, I am using native Keras layers wherever possible, which greatly simplified the code.  I have not used Lambda layers in this solution as other solutions appear more simpler/natural to me.

The biggest motivation for me to write this was to understand how exactly it is that these tensors flow through their lifecycle.  I think some of these concepts could be better represented in the class.  This is my first pull request! :) :) :)

### CODE REVIEW

**Comment by pritamdodeja** [comment]
In the latest commit, I have added a Lambda layer in-line with the original learning objectives.  The amount of data used for the lab associated with this notebook should be increased to avoid overfitting.  Since the approach I have implemented uses embeddings, it may be a good place to introduce the concept of sparsity and embeddings with a real-world example.
